unit_test/main_tests/main_options_test.py

```diff
@@ -29,14 +30,16 @@ def test_output_dir(platform, intercepted_build_args, monkeypatch):
     assert intercepted_build_args.args[0].globals.output_dir == OUTPUT_DIR.resolve()
 
 
-def test_output_dir_default(platform, intercepted_build_args, monkeypatch):
+@pytest.mark.usefixtures("platform")
+def test_output_dir_default(intercepted_build_args):
```

Comment:
The recommendation was to name the fixtures with an underscore if they don't return anything. I instead used this method.

Reply:
I like this